cmd/bbolt/main.go

```diff
@@ -59,12 +59,25 @@ func main() {
 	m := NewMain()
 	if err := m.Run(os.Args[1:]...); err == ErrUsage {
 		os.Exit(2)
+	} else if err == ErrUnknownCommand {
+		execute()
```

Comment:
Is it planned to migrate existing commands to Cobra in future?

Reply:
YES, see my comment above. We can work together to get it done.  cc @tjungblu as well